bin/docker/k8s/legacy-ingress/countly-ingress.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: countly-tls
+data:
+  tls.crt: YOUR_SSL_CERT_IN_BASE64      # cat cert.crt | base64
+  tls.key: YOUR_SSL_KEY_IN_BASE64       # cat cert.key | base64
+type: kubernetes.io/tls
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: countly-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /
+spec:
+  tls:
+  - hosts:
+    - YOUR_HOSTNAME                     # countly.example.com
```

Comment:
There is a lack of consistency for this requirement; the placeholder values I saw for the same parameter:
 - `<required domain name>`
 - `<required domain name>>`
 - `YOUR_HOST`

IMHO `YOUR_HOST` was okay. And why `example.count.ly` instead of `countly.example.com` as a comment? Second example is valid for non-Countly cloud customers.

Reply:
haven't changed anything in this file, just copied the file to legacy folder.

Reply:
my bad :) I pasted same text whenever I saw hosts thing :D